lib/elixir/lib/range.ex

```diff
@@ -70,25 +70,20 @@ defimpl Enumerable, for: Range do
     reduce(first, last, acc, fun, _up? = last >= first)
   end
 
-  defp reduce(_x, _y, {:halt, acc}, _fun, _up?) do
-    {:halted, acc}
-  end
```

Comment:
Please avoid changing the syntax. If the issue is about renaming variables, then it should keep the other syntax the same.